src/Nethermind/Nethermind.Core/JsonConverters/NullableByteReadOnlyMemoryConverter.cs

```diff
@@ -5,9 +5,9 @@
 using System.Text.Json;
 using System.Text.Json.Serialization;
 
-namespace Nethermind.Serialization.Json;
+namespace Nethermind.Core.JsonConverters;
```

Comment:
Why move projects and namespaces?

Reply:
Seems like those converters were moved to Core to solve circular dependency, so they'd got namespace not matching core project. I've just grouped them into a directory inside Core for convenience and changed namespace to match the structure. But I see the point, which is to have them all in one namespace. Is it critical? Seems like it creates unnecessary convention

Reply:
Anyway moved back so people will not lose converters bc of an additional ns